keras/layers/core.py

```diff
@@ -589,6 +589,27 @@ def get_config(self):
                 "output_reconstruction": self.output_reconstruction}
 
 
+class Maxout(Layer):
+    '''
+        Max-out layer, nb_feature is the number of pieces in the piecewise linear approx.
+        nb_feature must divide the first dimension of the input.
+    '''
+    def __init__(self, nb_feature=4):
+        super(Maxout, self).__init__()
+        self.nb_feature = int(nb_feature)
+
+    def get_output(self, train=False):
+        X = self.get_input(train)
+        nshape = T.concatenate([(X.shape[0], self.nb_feature, X.shape[1] // self.nb_feature),
```

Comment:
A shape being not a tensor but merely a tuple, why instantiate it via `T.concatenate` instead of simple tuple addition?